icicle-compiler/src/Icicle/Sea/FromAvalanche/Base.hs

```diff
@@ -91,7 +91,7 @@ mangleToSeaName = SeaName . Text.pack . concatMap mangle . show . pretty
      | c >= 'A' && c <= 'Z'
      = [c]
      | c == '_'
-     = "__"
+     = "_"
```

Comment:
ok, I guess it's also a good way to make sure all generated identifiers go through `mangle` (otherwise some tests will get mismatched names, e.g. `iint_t foo_bar; .... foo__bar = 0`)